src/llmq/quorums_signing_shares.cpp

```diff
@@ -928,6 +1014,43 @@ void CSigSharesManager::CollectSigSharesToSend(std::unordered_map<NodeId, std::u
     }
 }
 
+void CSigSharesManager::CollectSigSharesToSend(std::unordered_map<NodeId, std::vector<CSigShare>>& sigSharesToSend, const std::vector<CNode*>& vNodes)
+{
+    AssertLockHeld(cs);
+
+    std::unordered_map<uint256, CNode*> proTxToNode;
+    for (const auto& pnode : vNodes) {
+        if (pnode->verifiedProRegTxHash.IsNull()) {
+            continue;
+        }
+        proTxToNode.emplace(pnode->verifiedProRegTxHash, pnode);
+    }
+
+    auto curTime = GetAdjustedTime();
```

Comment:
that's what I had in the beginning, but it made testing very hard as `GetTime()` can't be mocked. Look for `# Let 1 second pass so that the next node is used for recovery, which should succeed` in `llmq-signing.py`

Reply:
My bad, mixed it up with `GetTimeMillis()`. Will fix now.